### PR TITLE
feat(navbar): add notification placeholder translations

### PR DESCRIPTION
**Pull request by JDownloader**

### CODE REVIEW

File: client/components/navbar/navbar-notification.service.js [hunk 1]
```diff
@@ -21,33 +21,31 @@ class NavbarNotificationService {
   getSubLinks() {
     return this.getMessages()
       .then(messages => messages.map(message => this.convertSubLink(message)))
-      .catch(() => undefined);
+      .catch(() => this.$q.when([]));
```
Comment (JDownloader) [inline client/components/navbar/navbar-notification.service.js]: this would not display en error message and you don't have to return a promise inside a promise, it's how chaning works.

File: client/components/navbar/navbar-notification.service.js [hunk 2]
```diff
@@ -21,33 +21,31 @@ class NavbarNotificationService {
...
-  /* eslint-disable */
-  _formatTime(dateTime) {
+  static _formatTime(dateTime) {
     return moment(dateTime).fromNow();
   }
 
   _toggleSublinkAction(toUpdate, linkClicked) {
     if (toUpdate.isActive && !toUpdate.updating) {
-      toUpdate.updating = true;
+      _.set(toUpdate, 'updating', true);
```
Comment (JDownloader) [inline client/components/navbar/navbar-notification.service.js]: this was not part of the PR, it's code that have been already been reviewed.

File: client/components/navbar/navbar-notification.service.js [hunk 3]
```diff
@@ -21,33 +21,31 @@ class NavbarNotificationService {
...
     } else if (!toUpdate.isActive && !toUpdate.updating && !linkClicked) {
-      toUpdate.updating = true;
+      _.set(toUpdate, 'updating', true);
       this.OvhApiNotificationAapi.post({ acknowledged: [toUpdate.id] }).$promise.then(() => {
-        toUpdate.isActive = true;
-        toUpdate.acknowledged = true;
-      }).finally(() => { toUpdate.updating = false; });
+        _.set(toUpdate, 'isActive', true);
+        _.set(toUpdate, 'acknowledged', true);
+      }).finally(() => { _.set(toUpdate, 'updating', false); });
```
Comment (neolitec) [inline client/components/navbar/navbar-notification.service.js]: Why so many _.set?

Comment (antleblanc) [reply client/components/navbar/navbar-notification.service.js]: due to: https://eslint.org/docs/rules/no-param-reassign

File: client/components/navbar/navbar-notification.service.js (hunk 2 above)
Comment (antleblanc) [reply client/components/navbar/navbar-notification.service.js]: Related to a required rebase. (see: https://github.com/ovh-ux/ovh-manager-telecom/pull/741#discussion_r199218411)
